src/formatDate.js

```diff
@@ -50,7 +50,42 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateFrom = date.split(fromFormat[3]);
+  const newDate = [];
+
+  const dayFormat = 'DD';
+  const monthFormat = 'MM';
+  const yearFormat = ['YY', 'YYYY'];
+
+  const day = fromFormat.indexOf(dayFormat);
```

Comment:
These variables store index, so correct name in this case would be smth like `dayIndex`